src/blob_gc.h

```diff
@@ -57,7 +69,8 @@ class BlobGC {
 
 struct GCScore {
   uint64_t file_number;
-  double score;
+  double gc_score;
```

Comment:
The type of double is narrowing conversion with blob_storage.cc:126.

Reply:
fixed